app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: None
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.skill_level = skill_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf blacksmith "
+                f"{self.nickname} with skill of the "
+                f"{self.skill_level} level")
+
+    def get_rating(self) -> None:
```

Comment:
The `get_rating` method should have a return type hint of `int` instead of `None`, as it returns `self.skill_level`, which is expected to be an integer.